rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -32,11 +32,13 @@
                 </or>
             </when>
             <perform>
-                <hint title="Java APIs for local file system" effort="1" category-id="optional">
-                    <message>Java APIs for local file system identified in the file.
-                    
-                    Local files in containers are not permanent storage and can disappear when an instance of the application terminates. Avoid accessing local filesystems from applications running in OpenShift environments, and consider using a database or cloud-based storage system instead.
+                <hint title="Java APIs for local file system" effort="1" category-id="openshift-potential">
```

Comment:
Oh! You are right. Looking at this, I did not saw it: https://github.com/mrizzi/windup-rulesets/blob/9e85f65022b389a06dff05a68cd5b6d5456002ec/rules-reviewed/core.windup.categories.xml

To me, "potential" categories do not make sense to me regarding cloud-readiness.

"Migration-potential" items are reflecting patterns in applications where "we know that we don't know". For example catch-alls. Such items have neither a clear recommendation, nor a meaningful level of effort.